server/src/test/java/org/elasticsearch/cluster/coordination/CoordinatorTests.java

```diff
@@ -115,6 +154,9 @@ private static String nodeIdFromIndex(int nodeIndex) {
             Settings.builder().put(NODE_NAME_SETTING.getKey(), "deterministic-task-queue").build());
         private final VotingConfiguration initialConfiguration;
 
+        private final Set<String> disconnectedNodes = new HashSet<>();
+        private final Set<String> partitionedNodes = new HashSet<>();
```

Comment:
I find `partitionedNodes` confusing, as I consider being disconnected just as a special case of being partitioned. I would prefer `blackholedNodes` here.

Reply:
Ok, I pushed 857013f9310386938b3e33e09933347cb314b19d.

My reasoning is that a partition is normally a blackhole; being disconnected requires a response indicating the lack of a remote listener, which is a different situation, but this is more consistent.